packages/analytics-js/src/components/eventManager/RudderEventFactory.ts

```diff
@@ -74,13 +72,20 @@ class RudderEventFactory {
    * @param from Old user ID
    * @param options API options
    */
-  private static generateAliasEvent(from?: string, options?: Nullable<ApiOptions>): RudderEvent {
+  private static generateAliasEvent(
+    to: string,
```

Comment:
these are Nullable< string > I believe

Reply:
@saikumarrs, pull the changes from v3-staging

Reply:
Addressed.
Since `previousId` will always have a value (default is anonymous ID), the type is set as `string | undefined`.

`to` is made `Nullable`.